metadata-ingestion/src/datahub/ingestion/source/mlflow.py

```diff
@@ -659,6 +850,18 @@ def _get_global_tags_workunit(
         )
         return wu
 
+    def _is_valid_platform(self, platform: Optional[str]) -> bool:
+        """Check if platform is registered as a source plugin"""
+        return platform in KNOWN_VALID_PLATFORM_NAMES
+
+    def _materialize_dataset_inputs(self) -> Optional[bool]:
+        """Check if dataset materialization is enabled"""
+        return self.config.materialize_dataset_inputs
+
+    def _source_type_to_platform_dict(self) -> Optional[dict]:
+        """Map MLflow source type to DataHub platform"""
+        return self.config.source_mapping_to_platform
+
```

Comment:
these two methods are not really necessary - we can just access `self.config.field` directly